Code/Framework/AzFramework/AzFramework/API/ApplicationAPI.h

```diff
@@ -168,6 +168,11 @@ namespace AzFramework
 
         virtual void OnMobileApplicationWillTerminate() {}
         virtual void OnMobileApplicationLowMemoryWarning() {}
+        
```

Comment:
So this is fine for us, but it feels like the wrong place if we want to send this back to o3de? Id expect it in some communication gem tbh

Reply:
> So this is fine for us, but it feels like the wrong place if we want to send this back to o3de? Id expect it in some communication gem tbh

Agree that it needs to be better, but this "the" home for the mobile/application API .. things.  Since Apple sends the notifications through the UIApplicationDelegate, thie ApplicationAPI is (currently) the right place to house it.